.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.words": [
+        "Stful",
+        "Stless"
+    ]
```

Comment:
What are these for?

Reply:
Oh sorry, it was because of the spell checker removing it.

Reply:
Great!